docs/compatibility.md

```diff
@@ -88,5 +88,10 @@ This would be indicated by a compiler error specific to a particular feature.
 * New core library versions may or may not require Kotlin compiler plugin update,
 depending on a particular release.
 We'll try to avoid these situations; however, in case of some unexpected issues, it may be necessary.
-So it is possible to have a situation where upgrading serialization runtime from `1.x` to `1.y` requires an update of Kotlin version from `1.4.0` to `1.4.x`.
+So it is possible to have a situation where upgrading serialization runtime from `1.x` to `1.y` requires an update of Kotlin version from `1.4.0` to `1.x.y`.
```

Comment:
Can I use other variables? 
With this writing, it seems like you need to use compiler `1.7.8` to switch from `1.7` to `1.8`.